_posts/2025-01-29-strimzicon2025-announcement.md

```diff
@@ -0,0 +1,68 @@
+---
+layout: post
+title:  "Welcome StrimziCon 2025!"
+date: 2025-01-29
+author: paolo_patierno
+---
+
+We are very pleased to announce the return of StrimziCon, scheduled later this year!
+After the huge success of [StrimziCon 2024](https://strimzi.io/blog/2024/06/06/strimzicon2024-roundup/), the Strimzi community decided to repeat the event for 2025.
+Like last year, this is going to be focused on our beloved Strimzi and Apache Kafka projects and it will be "virtual".
+It is organized and driven by the project maintainers in collaboration with the [Cloud Native Computing Foundation](https://www.cncf.io/).
+
+<!--more-->
+
+![StrimziCon 2025 Banner](/assets/images/posts/2025-01-29-strimzicon2025-banner.png)
+
+### Why attend StrimziCon?
+
+StrimziCon is an event for developers, DevOps engineers, and solution architects who want to learn about Strimzi and event streaming.
+This event will bring together the Strimzi community, and you will learn from experts and users about use cases, best practices, and their experience using Strimzi in their organizations.
+If you are working with cloud-native technologies, and you are looking for how to deal with data in motion in a cloud-based environment, don’t miss the opportunity.
+
+The conference will take place on <b>Wednesday, June 4th</b> (afternoon CEST), with one track covering the following themes:
+
+* Features, How Strimzi Works (features)
+* Operations, Using Strimzi with other projects (operations)
```

Comment:
```suggestion
The conference will take place on <b>Wednesday, June 4th</b> (afternoon CEST).
```

I think we should remove the part saying "with one track covering...". I don't think we need to advertise that it's a single track and themes wise the topics should be covered in the list below where we talk about the call for papers.